CrowServe/CrownLinkProtocol.h

```diff
@@ -33,11 +33,13 @@ struct ConnectionRequest : Header {
     u32 product_id;
     u32 version_id;
     u32 crownlink_version;
+    inline MessageType type() { return MessageType::ConnectionRequest; }
```

Comment:
This method can be marked const (after close paren and before open bracket).

I've been told inline is useless for inlined class accessors, but doesn't matter.